homeassistant/components/sensor/geizhals.py

```diff
@@ -80,6 +80,8 @@ def state(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
+        while (len(self.data.prices) < 4):
+            self.data.prices.append("None")
```

Comment:
Do we always want to show four prices exactly or could we just iterate available prices and show the ones available? Something like:
```py
attrs.update({
    'price{}'.format(idx + 1): price
    for idx, price in enumerate(self.data.prices)})
```